spec/models/question_spec.rb

```diff
@@ -54,12 +52,17 @@
   describe "#generate_answers" do
     let(:question) { Fabricate(:question) }
 
-    it "generates 4 answers total" do
+    it "generates 4 answers from mix" do
       Fabricate.times(9, :answer, question: question, correct: false)
       correct_answer = Fabricate(:answer, question: question, correct: true)
       expect(question.generate_answers.size).to eq(4)
     end
 
+    it "generates 4 answers even if all correct" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.